### PR TITLE
22w06a stuff

### PR DESCRIPTION
**Pull request by NoComment1105**
PR for mapping all unmapped classes that came along in 22w06a.

### CODE REVIEW

File: mappings/net/minecraft/client/PeriodicNotificationManager.mapping [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+	FIELD f_twnzglcs notificationTask Lnet/minecraft/unmapped/C_bianxjxy$C_fvgacjvq;
+	METHOD <init> (Lnet/minecraft/unmapped/C_ncpywfca;Lit/unimi/dsi/fastutil/objects/Object2BooleanFunction;)V
+		ARG 1 identifier
+		ARG 2 object2BooleanFunction
+	METHOD close close ()V
+	METHOD m_gwgfdxny calculateInitialDelay (Ljava/util/List;)J
+	METHOD m_rocjzhkk stopTimer ()V
+	METHOD m_vygiodom calculateOptimalPeriod (Ljava/util/List;J)J
+	CLASS C_fvgacjvq NotificationTask
+		FIELD f_jnnycmjg instance Lnet/minecraft/unmapped/C_ayfeobid;
```
Comment (OroArmor) [inline mappings/net/minecraft/client/PeriodicNotificationManager.mapping]:
```suggestion
		FIELD f_jnnycmjg client Lnet/minecraft/unmapped/C_ayfeobid;
```

File: mappings/net/minecraft/datafixer/fix/ChunkBedBlockEntityFix.mapping [hunk 2]
```diff
@@ -0,0 +1,2 @@
+CLASS net/minecraft/unmapped/C_hqgjioon net/minecraft/datafixer/fix/ChunkBedBlockEntityFix
+	METHOD m_dmtypzij cap (Lcom/mojang/datafixers/types/Type;Lcom/mojang/datafixers/types/templates/List$ListType;)Lcom/mojang/datafixers/TypeRewriteRule;
```
Comment (OroArmor) [inline mappings/net/minecraft/datafixer/fix/ChunkBedBlockEntityFix.mapping]:
```suggestion
	METHOD m_dmtypzij fixBedBlockEntity (Lcom/mojang/datafixers/types/Type;Lcom/mojang/datafixers/types/templates/List$ListType;)Lcom/mojang/datafixers/TypeRewriteRule;
```

File: mappings/net/minecraft/resource/packs/CloseableResourceManager.mapping [hunk 3]
```diff
@@ -0,0 +1,2 @@
+CLASS net/minecraft/unmapped/C_carvbxzj net/minecraft/resource/packs/CloseableResourceManager
```
Comment (OroArmor) [inline mappings/net/minecraft/resource/packs/CloseableResourceManager.mapping]:
```suggestion
CLASS net/minecraft/unmapped/C_carvbxzj net/minecraft/resource/packs/AutoCloseableResourceManager
```

**Review by OroArmor — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/resource/packs/CloseableResourceManager.mapping [hunk 4]
```diff
@@ -1,2 +1,2 @@
-CLASS net/minecraft/unmapped/C_carvbxzj net/minecraft/resource/packs/CloseableResourceManager
+CLASS net/minecraft/unmapped/C_carvbxzj net/minecraft/resource/packs/AutoCloseableResourceManager
```
Comment (OroArmor) [inline mappings/net/minecraft/resource/packs/CloseableResourceManager.mapping]: Didnt change the file name

Comment (NoComment1105) [reply mappings/net/minecraft/resource/packs/CloseableResourceManager.mapping]: Heck this

Comment (NoComment1105) [reply mappings/net/minecraft/resource/packs/CloseableResourceManager.mapping]: Think it's done now

**Review by OroArmor — APPROVED** [review]

**Comment by NoComment1105** [comment]
This PR is no now longer a WIP one. You're free to full review

File: mappings/net/minecraft/resource/packs/MultiPackResourceManager.mapping [hunk 5]
```diff
@@ -0,0 +1,3 @@
+CLASS net/minecraft/unmapped/C_xzrqjrss net/minecraft/resource/packs/MultiPackResourceManager
```
Comment (LambdAurora) [inline mappings/net/minecraft/resource/packs/MultiPackResourceManager.mapping]: Quilt Mapping convention: no plural package name.

Also we don't have any dedicated `pack` package, for now put it in `resource`, if we want to subpackage it should be in a separate refactor PR.

File: mappings/net/minecraft/server/WorldStem.mapping [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+		FIELD f_mwtkgtsb commandSelection Lnet/minecraft/unmapped/C_rieemtgi$C_vypaxkxj;
+		FIELD f_sdogjwod resourcePackManager Lnet/minecraft/unmapped/C_dqfyacfy;
+		FIELD f_uiersxov safeMode Z
+		METHOD <init> (Lnet/minecraft/unmapped/C_dqfyacfy;Lnet/minecraft/unmapped/C_rieemtgi$C_vypaxkxj;IZ)V
+			ARG 1 resourcePackManager
+			ARG 2 commandSelection
+			ARG 3 level
+			ARG 4 safeMode
+		METHOD m_cxcmncyf safeMode ()Z
+		METHOD m_dgnfjecd reosurcePackManager ()Lnet/minecraft/unmapped/C_dqfyacfy;
```
Comment (LambdAurora) [inline mappings/net/minecraft/server/WorldStem.mapping]:
```suggestion
		METHOD m_dgnfjecd resourcePackManager ()Lnet/minecraft/unmapped/C_dqfyacfy;
```

**Review by LambdAurora — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/resource/packs/MultiPackResourceManager.mapping (hunk 5 above)
Comment (NoComment1105) [reply mappings/net/minecraft/resource/packs/MultiPackResourceManager.mapping]: done

**Review by LambdAurora — APPROVED** [review]

File: mappings/net/minecraft/server/ServerReloadableResources.mapping [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+	METHOD m_qobkrmhz listeners ()Ljava/util/List;
+	METHOD m_qpiprbld getLootFunctions ()Lnet/minecraft/unmapped/C_wqgroktp;
+	METHOD m_qzzzeens getRecipes ()Lnet/minecraft/unmapped/C_hjseusrb;
+	METHOD m_ravmxbhg updateRegistryTags (Lnet/minecraft/unmapped/C_wqxmvzdq;)V
+	METHOD m_txfqcvol getLootConditions ()Lnet/minecraft/unmapped/C_gztbwabg;
+	METHOD m_xlzbyznn loadResources (Lnet/minecraft/unmapped/C_tmnrpasf;Lnet/minecraft/unmapped/C_wqxmvzdq$C_ggmzysrd;Lnet/minecraft/unmapped/C_rieemtgi$C_vypaxkxj;ILjava/util/concurrent/Executor;Ljava/util/concurrent/Executor;)Ljava/util/concurrent/CompletableFuture;
+		ARG 0 resources
+		ARG 2 environment
+		ARG 4 executor
+		ARG 5 executor2
```
Comment (OroArmor) [inline mappings/net/minecraft/server/ServerReloadableResources.mapping]: i would try to figure more about what these do. if there are just two then this is fine.

**Review by OroArmor — APPROVED** [review]

File: mappings/net/minecraft/server/ServerReloadableResources.mapping (hunk 7 above)
Comment (NoComment1105) [reply mappings/net/minecraft/server/ServerReloadableResources.mapping]: I'll look into it later o7

File: mappings/net/minecraft/client/PeriodicNotificationManager.mapping [hunk 8]
```diff
@@ -0,0 +1,30 @@
+CLASS net/minecraft/unmapped/C_bianxjxy net/minecraft/client/PeriodicNotificationManager
+	FIELD f_dlgrskix LOGGER Lorg/slf4j/Logger;
+	FIELD f_gouzfydb timer Ljava/util/Timer;
+	FIELD f_ikmicyau selector Lit/unimi/dsi/fastutil/objects/Object2BooleanFunction;
+	FIELD f_jliiyfor CODEC Lcom/mojang/serialization/Codec;
+	FIELD f_jzmobybo notifications Lnet/minecraft/unmapped/C_ncpywfca;
+	FIELD f_twnzglcs notificationTask Lnet/minecraft/unmapped/C_bianxjxy$C_fvgacjvq;
+	METHOD <init> (Lnet/minecraft/unmapped/C_ncpywfca;Lit/unimi/dsi/fastutil/objects/Object2BooleanFunction;)V
+		ARG 1 identifier
+		ARG 2 object2BooleanFunction
```
Comment (IotaBread) [inline mappings/net/minecraft/client/PeriodicNotificationManager.mapping]: What is this for?

File: mappings/net/minecraft/client/PeriodicNotificationManager.mapping [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+	METHOD m_rocjzhkk stopTimer ()V
+	METHOD m_vygiodom calculateOptimalPeriod (Ljava/util/List;J)J
+	CLASS C_fvgacjvq NotificationTask
+		FIELD f_jnnycmjg client Lnet/minecraft/unmapped/C_ayfeobid;
+		FIELD f_lifjeken notifications Ljava/util/List;
+		FIELD f_oirwnkjq period J
+		FIELD f_shihovkr elapsed Ljava/util/concurrent/atomic/AtomicLong;
+		METHOD m_eczutovu reset (Ljava/util/List;J)Lnet/minecraft/unmapped/C_bianxjxy$C_fvgacjvq;
+		METHOD run run ()V
+	CLASS C_puurieil Notification
```
Comment (IotaBread) [inline mappings/net/minecraft/client/PeriodicNotificationManager.mapping:26]: Could you add the `equals(Object)` parameter name `o`?

File: mappings/net/minecraft/client/PeriodicNotificationManager.mapping [hunk 10]
```diff
@@ -0,0 +1,30 @@
...
+		ARG 2 object2BooleanFunction
+	METHOD close close ()V
+	METHOD m_gwgfdxny calculateInitialDelay (Ljava/util/List;)J
+	METHOD m_rocjzhkk stopTimer ()V
+	METHOD m_vygiodom calculateOptimalPeriod (Ljava/util/List;J)J
+	CLASS C_fvgacjvq NotificationTask
+		FIELD f_jnnycmjg client Lnet/minecraft/unmapped/C_ayfeobid;
+		FIELD f_lifjeken notifications Ljava/util/List;
+		FIELD f_oirwnkjq period J
+		FIELD f_shihovkr elapsed Ljava/util/concurrent/atomic/AtomicLong;
```
Comment (IotaBread) [inline mappings/net/minecraft/client/PeriodicNotificationManager.mapping]: `elapsedTime` maybe?

File: mappings/net/minecraft/client/particle/BlockMarker.mapping [hunk 11]
```diff
@@ -0,0 +1,8 @@
+CLASS net/minecraft/unmapped/C_uygjicct net/minecraft/client/particle/BlockMarker
```
Comment (IotaBread) [inline mappings/net/minecraft/client/particle/BlockMarker.mapping]: Already mapped in #68

File: mappings/net/minecraft/datafixer/fix/ChunkBedBlockEntityFix.mapping [hunk 12]
```diff
@@ -0,0 +1,2 @@
+CLASS net/minecraft/unmapped/C_hqgjioon net/minecraft/datafixer/fix/ChunkBedBlockEntityFix
```
Comment (IotaBread) [inline mappings/net/minecraft/datafixer/fix/ChunkBedBlockEntityFix.mapping]: Already mapped in #68

File: mappings/net/minecraft/datafixer/fix/StructureSettingsFlattenFix.mapping [hunk 13]
```diff
@@ -0,0 +1,6 @@
+CLASS net/minecraft/unmapped/C_pizyerbz net/minecraft/datafixer/fix/StructureSettingsFlattenFix
```
Comment (IotaBread) [inline mappings/net/minecraft/datafixer/fix/StructureSettingsFlattenFix.mapping]: `FlattenStructureSettingsFix`?

File: mappings/net/minecraft/entity/ai/goal/ClimbOnTopOfPowderSnowGoal.mapping [hunk 14]
```diff
@@ -0,0 +1,3 @@
+CLASS net/minecraft/unmapped/C_iyfwctmk net/minecraft/entity/ai/goal/ClimbOnTopOfPowderSnowGoal
+	FIELD f_brjbxqmk world Lnet/minecraft/unmapped/C_cdctfzbn;
+	FIELD f_neafhlre mobEntity Lnet/minecraft/unmapped/C_dxkfswlz;
```
Comment (IotaBread) [inline mappings/net/minecraft/entity/ai/goal/ClimbOnTopOfPowderSnowGoal.mapping]:
```suggestion
	FIELD f_neafhlre mob Lnet/minecraft/unmapped/C_dxkfswlz;
```

File: mappings/net/minecraft/resource/HolderSetCodec.mapping [hunk 15]
```diff
@@ -0,0 +1,16 @@
+CLASS net/minecraft/unmapped/C_wbhfljwo net/minecraft/resource/HolderSetCodec
+	FIELD f_kzoidxjr registryAwareCodec Lcom/mojang/serialization/Codec;
+	FIELD f_pbpbaevc registryKey Lnet/minecraft/unmapped/C_xhhleach;
```
Comment (IotaBread) [inline mappings/net/minecraft/resource/HolderSetCodec.mapping]: This registryKey is used to reference the registry used by the `HolderSet`, so the name could be `registry`

File: mappings/net/minecraft/resource/HolderSetCodec.mapping [hunk 16]
```diff
@@ -0,0 +1,16 @@
...
+	FIELD f_qgzyttcp elementCodec Lcom/mojang/serialization/Codec;
+	FIELD f_uouarqmt homogeneousListCodec Lcom/mojang/serialization/Codec;
+	METHOD <init> (Lnet/minecraft/unmapped/C_xhhleach;Lcom/mojang/serialization/Codec;Z)V
+		ARG 1 registryKey
+		ARG 2 elementCodec
+	METHOD decode decode (Lcom/mojang/serialization/DynamicOps;Ljava/lang/Object;)Lcom/mojang/serialization/DataResult;
+		ARG 1 dynamicOps
+	METHOD encode encode (Ljava/lang/Object;Lcom/mojang/serialization/DynamicOps;Ljava/lang/Object;)Lcom/mojang/serialization/DataResult;
+		ARG 2 dynamicOps
+	METHOD m_cjfthajd homoGeneousList (Lcom/mojang/serialization/Codec;Z)Lcom/mojang/serialization/Codec;
```
Comment (IotaBread) [inline mappings/net/minecraft/resource/HolderSetCodec.mapping]:
```suggestion
	METHOD m_cjfthajd homogeneousList (Lcom/mojang/serialization/Codec;Z)Lcom/mojang/serialization/Codec;
```

File: mappings/net/minecraft/resource/MultiPackResourceManager.mapping [hunk 17]
```diff
@@ -0,0 +1,3 @@
+CLASS net/minecraft/unmapped/C_xzrqjrss net/minecraft/resource/MultiPackResourceManager
+	FIELD f_haylgaaj packs Ljava/util/List;
+	FIELD f_srpgcayr namespacedManager Ljava/util/Map;
```
Comment (IotaBread) [inline mappings/net/minecraft/resource/MultiPackResourceManager.mapping]:
```suggestion
	FIELD f_srpgcayr namespaceManagers Ljava/util/Map;
```

File: mappings/net/minecraft/server/ServerReloadableResources.mapping [hunk 18]
```diff
@@ -0,0 +1,26 @@
+CLASS net/minecraft/unmapped/C_migzkpst net/minecraft/server/ServerReloadableResources
+	FIELD f_bqclfysi LOGGER Lorg/slf4j/Logger;
+	FIELD f_cqvnljsk recipes Lnet/minecraft/unmapped/C_hjseusrb;
+	FIELD f_izcsyrwb functionLibrary Lnet/minecraft/unmapped/C_coevuhwv;
```
Comment (IotaBread) [inline mappings/net/minecraft/server/ServerReloadableResources.mapping]: Why the library suffix here?

File: mappings/net/minecraft/server/WorldStem.mapping [hunk 19]
```diff
@@ -0,0 +1,36 @@
+CLASS net/minecraft/unmapped/C_tljyhzqp net/minecraft/server/WorldStem
+	FIELD f_hmwuaxvg worldData Lnet/minecraft/unmapped/C_jkevluio;
+	FIELD f_sphgqqzg resourceManager Lnet/minecraft/unmapped/C_carvbxzj;
+	FIELD f_tchbvinx registryAccess Lnet/minecraft/unmapped/C_wqxmvzdq$C_ggmzysrd;
+	FIELD f_twvealqc dataPackResources Lnet/minecraft/unmapped/C_migzkpst;
+	METHOD <init> (Lnet/minecraft/unmapped/C_carvbxzj;Lnet/minecraft/unmapped/C_migzkpst;Lnet/minecraft/unmapped/C_wqxmvzdq$C_ggmzysrd;Lnet/minecraft/unmapped/C_jkevluio;)V
+		ARG 1 resourceManager
+		ARG 2 dataPackResources
+		ARG 3 registryAccess
+		ARG 4 worldData
```
Comment (IotaBread) [inline mappings/net/minecraft/server/WorldStem.mapping]:
```suggestion
	FIELD f_hmwuaxvg saveProperties Lnet/minecraft/unmapped/C_jkevluio;
	FIELD f_sphgqqzg resourceManager Lnet/minecraft/unmapped/C_carvbxzj;
	FIELD f_tchbvinx registryManager Lnet/minecraft/unmapped/C_wqxmvzdq$C_ggmzysrd;
	FIELD f_twvealqc dataPackResources Lnet/minecraft/unmapped/C_migzkpst;
	METHOD <init> (Lnet/minecraft/unmapped/C_carvbxzj;Lnet/minecraft/unmapped/C_migzkpst;Lnet/minecraft/unmapped/C_wqxmvzdq$C_ggmzysrd;Lnet/minecraft/unmapped/C_jkevluio;)V
		ARG 1 resourceManager
		ARG 2 dataPackResources
		ARG 3 registryManager
		ARG 4 saveProperties
```

File: mappings/net/minecraft/server/WorldStem.mapping [hunk 20]
```diff
@@ -0,0 +1,36 @@
...
+	METHOD close close ()V
+	METHOD m_bgugujxe load (Lnet/minecraft/unmapped/C_tljyhzqp$C_wljmulvp;Lnet/minecraft/unmapped/C_tljyhzqp$C_orcvexst;Lnet/minecraft/unmapped/C_tljyhzqp$C_gwngutql;Ljava/util/concurrent/Executor;Ljava/util/concurrent/Executor;)Ljava/util/concurrent/CompletableFuture;
+	METHOD m_frmggyww updateGlobals ()V
+	METHOD m_pfrxpzhv dataPackResources ()Lnet/minecraft/unmapped/C_migzkpst;
+	METHOD m_vmpnykme resourceManager ()Lnet/minecraft/unmapped/C_carvbxzj;
+	METHOD m_vqetafdc worldData ()Lnet/minecraft/unmapped/C_jkevluio;
+	METHOD m_wyhjhabc registryAccess ()Lnet/minecraft/unmapped/C_wqxmvzdq$C_ggmzysrd;
```
Comment (IotaBread) [inline mappings/net/minecraft/server/WorldStem.mapping]:
```suggestion
	METHOD m_vqetafdc saveProperties ()Lnet/minecraft/unmapped/C_jkevluio;
	METHOD m_wyhjhabc registryManager ()Lnet/minecraft/unmapped/C_wqxmvzdq$C_ggmzysrd;
```

File: mappings/net/minecraft/server/WorldStem.mapping [hunk 21]
```diff
@@ -0,0 +1,36 @@
...
+	CLASS C_gwngutql WorldDataSupplier
+		METHOD get get (Lnet/minecraft/unmapped/C_tmnrpasf;Lnet/minecraft/unmapped/C_tjllhmjy;)Lcom/mojang/datafixers/util/Pair;
+		METHOD loadFromWorld loadFromWorld (Lnet/minecraft/unmapped/C_bpwagnit$C_unecclfw;)Lnet/minecraft/unmapped/C_tljyhzqp$C_gwngutql;
+	CLASS C_orcvexst Supplier
+		METHOD loadFromWorld loadFromWorld (Lnet/minecraft/unmapped/C_bpwagnit$C_unecclfw;)Lnet/minecraft/unmapped/C_tljyhzqp$C_orcvexst;
+	CLASS C_wljmulvp InitConfig
+		FIELD f_dipzoqzv functionCompilationLevel I
+		FIELD f_mwtkgtsb commandSelection Lnet/minecraft/unmapped/C_rieemtgi$C_vypaxkxj;
```
Comment (IotaBread) [inline mappings/net/minecraft/server/WorldStem.mapping]:
```suggestion
		FIELD f_mwtkgtsb commandRegistrationEnvironment Lnet/minecraft/unmapped/C_rieemtgi$C_vypaxkxj;
```

File: mappings/net/minecraft/server/WorldStem.mapping [hunk 22]
```diff
@@ -0,0 +1,36 @@
...
+		FIELD f_sdogjwod resourcePackManager Lnet/minecraft/unmapped/C_dqfyacfy;
+		FIELD f_uiersxov safeMode Z
+		METHOD <init> (Lnet/minecraft/unmapped/C_dqfyacfy;Lnet/minecraft/unmapped/C_rieemtgi$C_vypaxkxj;IZ)V
+			ARG 1 resourcePackManager
+			ARG 2 commandSelection
```
Comment (IotaBread) [inline mappings/net/minecraft/server/WorldStem.mapping]:
```suggestion
			ARG 2 commandRegistrationEnvironment
```

File: mappings/net/minecraft/server/WorldStem.mapping [hunk 23]
```diff
@@ -0,0 +1,36 @@
...
+			ARG 3 level
+			ARG 4 safeMode
+		METHOD m_cxcmncyf safeMode ()Z
+		METHOD m_dgnfjecd resourcePackManager ()Lnet/minecraft/unmapped/C_dqfyacfy;
+		METHOD m_odnpcamv commandSelection ()Lnet/minecraft/unmapped/C_rieemtgi$C_vypaxkxj;
```
Comment (IotaBread) [inline mappings/net/minecraft/server/WorldStem.mapping]:
```suggestion
		METHOD m_odnpcamv commandRegistrationEnvironment ()Lnet/minecraft/unmapped/C_rieemtgi$C_vypaxkxj;
```

File: mappings/net/minecraft/structure/StructurePlacement.mapping [hunk 24]
```diff
@@ -0,0 +1,4 @@
+CLASS net/minecraft/unmapped/C_dzcnonip net/minecraft/structure/StructurePlacement
+	FIELD f_kxfcknke CODEC Lcom/mojang/serialization/Codec;
+	METHOD m_luiblzbu isFeatureChunk (Lnet/minecraft/unmapped/C_kgfalbim;II)Z
```
Comment (IotaBread) [inline mappings/net/minecraft/structure/StructurePlacement.mapping]: `chunkHasFeature` maybe?

File: mappings/net/minecraft/structure/piece/PieceGeneratorSupplier.mapping [hunk 25]
```diff
@@ -0,0 +1,34 @@
+CLASS net/minecraft/unmapped/C_ukcqthce net/minecraft/structure/piece/PieceGeneratorSupplier
```
Comment (IotaBread) [inline mappings/net/minecraft/structure/piece/PieceGeneratorSupplier.mapping]: Already mapped in #68

File: mappings/net/minecraft/structure/processor/PostPlacementProcessor.mapping [hunk 26]
```diff
@@ -0,0 +1,3 @@
+CLASS net/minecraft/unmapped/C_kgyvgadg net/minecraft/structure/processor/PostPlacementProcessor
```
Comment (IotaBread) [inline mappings/net/minecraft/structure/processor/PostPlacementProcessor.mapping]: Already mapped in #68

File: mappings/net/minecraft/util/NativeModuleListener.mapping [hunk 27]
```diff
@@ -0,0 +1,20 @@
+CLASS net/minecraft/unmapped/C_frwmzmms net/minecraft/util/NativeModuleListener
```
Comment (IotaBread) [inline mappings/net/minecraft/util/NativeModuleListener.mapping]: Already mapped in #68

File: mappings/net/minecraft/util/profiling/jfr/Percentiles.mapping [hunk 28]
```diff
@@ -0,0 +1,5 @@
+CLASS net/minecraft/unmapped/C_fkzluywb net/minecraft/util/profiling/jfr/Percentiles
```
Comment (IotaBread) [inline mappings/net/minecraft/util/profiling/jfr/Percentiles.mapping]: Already mapped in #68

File: mappings/net/minecraft/util/profiling/jfr/SummaryReporter.mapping [hunk 29]
```diff
@@ -0,0 +1,6 @@
+CLASS net/minecraft/unmapped/C_icovikpg net/minecraft/util/profiling/jfr/SummaryReporter
```
Comment (IotaBread) [inline mappings/net/minecraft/util/profiling/jfr/SummaryReporter.mapping]: Already mapped in #68

File: mappings/net/minecraft/util/registry/RegistryLoader.mapping [hunk 30]
```diff
@@ -0,0 +1,34 @@
...
+		ARG 5 dynamicOps
+	METHOD m_fagrbqwd bind (Lnet/minecraft/unmapped/C_wqxmvzdq$C_qndozwiw;)Lnet/minecraft/unmapped/C_yftyvswu$C_ogvxrsre;
+	METHOD m_nvaacgyw readCache (Lnet/minecraft/unmapped/C_xhhleach;)Lnet/minecraft/unmapped/C_yftyvswu$C_gkioyedo;
+	METHOD m_zgvxgojh (Lnet/minecraft/unmapped/C_xhhleach;Lcom/mojang/serialization/Codec;Lnet/minecraft/unmapped/C_xhhleach;Lcom/mojang/serialization/DynamicOps;Lnet/minecraft/unmapped/C_eexxncvi;)Lcom/mojang/serialization/DataResult;
+		ARG 5 mutableRegistry
+	CLASS C_gkioyedo ReadCache
+		FIELD f_yolhkzjs values Ljava/util/Map;
+	CLASS C_ogvxrsre Bound
+		FIELD f_dgkxovrn loader Lnet/minecraft/unmapped/C_yftyvswu;
+		FIELD f_gfawiple access Lnet/minecraft/unmapped/C_wqxmvzdq$C_qndozwiw;
```
Comment (IotaBread) [inline mappings/net/minecraft/util/registry/RegistryLoader.mapping]:
```suggestion
		FIELD f_gfawiple manager Lnet/minecraft/unmapped/C_wqxmvzdq$C_qndozwiw;
```

File: mappings/net/minecraft/util/registry/RegistryLoader.mapping [hunk 31]
```diff
@@ -0,0 +1,34 @@
...
+		METHOD <init> (Lnet/minecraft/unmapped/C_wqxmvzdq$C_qndozwiw;Lnet/minecraft/unmapped/C_yftyvswu;)V
+			ARG 1 access
```
Comment (IotaBread) [inline mappings/net/minecraft/util/registry/RegistryLoader.mapping]:
```suggestion
			ARG 1 manager
```

File: mappings/net/minecraft/util/registry/RegistryLoader.mapping [hunk 32]
```diff
@@ -0,0 +1,34 @@
...
+			ARG 2 loader
+		METHOD m_iqsvzpdj loader ()Lnet/minecraft/unmapped/C_yftyvswu;
+		METHOD m_nutcxuqn access ()Lnet/minecraft/unmapped/C_wqxmvzdq$C_qndozwiw;
```
Comment (IotaBread) [inline mappings/net/minecraft/util/registry/RegistryLoader.mapping]:
```suggestion
		METHOD m_nutcxuqn manager ()Lnet/minecraft/unmapped/C_wqxmvzdq$C_qndozwiw;
```

File: mappings/net/minecraft/util/registry/RegistryOps.mapping [hunk 33]
```diff
@@ -0,0 +1,21 @@
+CLASS net/minecraft/unmapped/C_piadqmzm net/minecraft/util/registry/RegistryOps
+	FIELD f_cbvxdsia loader Ljava/util/Optional;
+	FIELD f_mqgrslrq registryAccess Lnet/minecraft/unmapped/C_wqxmvzdq;
```
Comment (IotaBread) [inline mappings/net/minecraft/util/registry/RegistryOps.mapping]:
```suggestion
	FIELD f_mqgrslrq registryManager Lnet/minecraft/unmapped/C_wqxmvzdq;
```

File: mappings/net/minecraft/util/registry/RegistryOps.mapping [hunk 34]
```diff
@@ -0,0 +1,21 @@
...
+	FIELD f_qiknxklx asJson Lcom/mojang/serialization/DynamicOps;
+	METHOD <init> (Lcom/mojang/serialization/DynamicOps;Lnet/minecraft/unmapped/C_wqxmvzdq;Ljava/util/Optional;)V
+		ARG 1 dynamicOps
+		ARG 2 registryAccess
```
Comment (IotaBread) [inline mappings/net/minecraft/util/registry/RegistryOps.mapping]:
```suggestion
		ARG 2 registryManager
```

File: mappings/net/minecraft/util/registry/RegistryOps.mapping [hunk 35]
```diff
@@ -0,0 +1,21 @@
...
+		ARG 2 resources
+	METHOD m_bvagnmeu jsonLoader ()Lcom/mojang/serialization/DynamicOps;
+	METHOD m_cqutychh createAndLoad (Lcom/mojang/serialization/DynamicOps;Lnet/minecraft/unmapped/C_wqxmvzdq$C_qndozwiw;Lnet/minecraft/unmapped/C_nyvkclsy;)Lnet/minecraft/unmapped/C_piadqmzm;
+		ARG 0 dynamicOps
+		ARG 2 resourceAccess
+	METHOD m_dfidsvas getRegistry (Lnet/minecraft/unmapped/C_xhhleach;)Lcom/mojang/serialization/MapCodec;
+	METHOD m_guxhmsde registryLoader ()Ljava/util/Optional;
+	METHOD m_zvayubze create (Lcom/mojang/serialization/DynamicOps;Lnet/minecraft/unmapped/C_wqxmvzdq;)Lnet/minecraft/unmapped/C_piadqmzm;
+		ARG 0 dynamicOps
+		ARG 1 registryAccess
```
Comment (IotaBread) [inline mappings/net/minecraft/util/registry/RegistryOps.mapping]:
```suggestion
		ARG 1 registryManager
```

File: mappings/net/minecraft/world/gen/blockpredicate/MatchingBlockTagPredicate.mapping [hunk 36]
```diff
@@ -0,0 +1,5 @@
+CLASS net/minecraft/unmapped/C_zkfeojkf net/minecraft/world/gen/blockpredicate/MatchingBlockTagPredicate
```
Comment (IotaBread) [inline mappings/net/minecraft/world/gen/blockpredicate/MatchingBlockTagPredicate.mapping]: Already mapped in #68

**Review by IotaBread — CHANGES_REQUESTED** [review]
Mostly some conflicts with #68 and incongruencies between our mappings and mojmap (RegistryManager vs RegistryAccess for example)

File: mappings/net/minecraft/datafixer/fix/StructureSettingsFlattenFix.mapping (hunk 13 above)
Comment (NoComment1105) [reply mappings/net/minecraft/datafixer/fix/StructureSettingsFlattenFix.mapping]: I'd agree

File: mappings/net/minecraft/structure/piece/PieceGeneratorSupplier.mapping (hunk 25 above)
Comment (NoComment1105) [reply mappings/net/minecraft/structure/piece/PieceGeneratorSupplier.mapping]: I knew I'd seen it somewhere >:|

File: mappings/net/minecraft/structure/processor/PostPlacementProcessor.mapping (hunk 26 above)
Comment (NoComment1105) [reply mappings/net/minecraft/structure/processor/PostPlacementProcessor.mapping:1]: Another one waaaa

File: mappings/net/minecraft/client/PeriodicNotificationManager.mapping (hunk 8 above)
Comment (NoComment1105) [reply mappings/net/minecraft/client/PeriodicNotificationManager.mapping]: I didn't change the name properly, but it's what gets applied to `selector` (f_ikmicyau). I'll fix that now

File: mappings/net/minecraft/resource/HolderSetCodec.mapping (hunk 15 above)
Comment (NoComment1105) [reply mappings/net/minecraft/resource/HolderSetCodec.mapping]: Ah I see. Sorted!

File: mappings/net/minecraft/server/ServerReloadableResources.mapping (hunk 18 above)
Comment (NoComment1105) [reply mappings/net/minecraft/server/ServerReloadableResources.mapping]: mistake

File: mappings/net/minecraft/server/ServerReloadableResources.mapping [hunk 37]
```diff
@@ -0,0 +1,26 @@
...
+	METHOD m_qzzzeens getRecipes ()Lnet/minecraft/unmapped/C_hjseusrb;
+	METHOD m_ravmxbhg updateRegistryTags (Lnet/minecraft/unmapped/C_wqxmvzdq;)V
+	METHOD m_txfqcvol getLootConditions ()Lnet/minecraft/unmapped/C_gztbwabg;
+	METHOD m_xlzbyznn loadResources (Lnet/minecraft/unmapped/C_tmnrpasf;Lnet/minecraft/unmapped/C_wqxmvzdq$C_ggmzysrd;Lnet/minecraft/unmapped/C_rieemtgi$C_vypaxkxj;ILjava/util/concurrent/Executor;Ljava/util/concurrent/Executor;)Ljava/util/concurrent/CompletableFuture;
+		ARG 0 resources
+		ARG 2 environment
+		ARG 4 prepareExecutor
+		ARG 5 applyExecutor
+	METHOD m_ygmlfrda getLootTables ()Lnet/minecraft/unmapped/C_chzdbgdx;
+	METHOD m_zxbtczuz getFunctionLibrary ()Lnet/minecraft/unmapped/C_coevuhwv;
```
Comment (IotaBread) [inline mappings/net/minecraft/server/ServerReloadableResources.mapping]:
```suggestion
	METHOD m_zxbtczuz getFunctions ()Lnet/minecraft/unmapped/C_coevuhwv;
```

File: mappings/net/minecraft/util/registry/RegistryLoader.mapping [hunk 38]
```diff
@@ -0,0 +1,34 @@
+CLASS net/minecraft/unmapped/C_yftyvswu net/minecraft/util/registry/RegistryLoader
+	FIELD f_usmthubw readCache Ljava/util/Map;
+	FIELD f_vmooysue registryAccess Lnet/minecraft/unmapped/C_nyvkclsy;
```
Comment (IotaBread) [inline mappings/net/minecraft/util/registry/RegistryLoader.mapping]:
```suggestion
	FIELD f_vmooysue registryManager Lnet/minecraft/unmapped/C_nyvkclsy;
```

File: mappings/net/minecraft/util/registry/RegistryLoader.mapping [hunk 39]
```diff
@@ -0,0 +1,34 @@
...
+	METHOD <init> (Lnet/minecraft/unmapped/C_nyvkclsy;)V
+		ARG 1 registryAccess
```
Comment (IotaBread) [inline mappings/net/minecraft/util/registry/RegistryLoader.mapping]:
```suggestion
		ARG 1 registryManager
```

**Review by IotaBread — APPROVED** [review]

**Pull request merged**